backend/infrastructures/apps/extract/views/views.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from pathlib import Path
+from typing import cast
+
+import inject
+from django.core.files.storage import FileSystemStorage
+from pandera.typing.pandas import DataFrame
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from modules.common.domain import exceptions as domain_exceptions
+from modules.data.domain import value_objects as data_value_objects
+from modules.extract.domain import commands as domain_extract_commands
+from modules.extract.services import commands as service_extract_commands
+from modules.load.domain import commands as domain_load_commands
+from modules.load.domain.ports import units_of_work
+from modules.load.services import commands as services_load_commands
+from modules.load.services import queries as load_queries
+from modules.transform.domain import commands as domain_transform_commands
+from modules.transform.services import commands as services_transform_commands
+
+from ....config import settings
+
+logger = logging.getLogger(__name__)
+
+
+class ExtractViewSet(
+    ViewSet,
+):
+    @inject.param(name="save_data_unit_of_work", cls="save_data_unit_of_work")
+    def create(
+        self,
+        request: Request,
+        save_data_unit_of_work: units_of_work.AbstractDataUnitOfWork,
+    ) -> Response:
+        if "file" not in request.FILES:
+            return Response(
+                data={"error": "File is not attached to the request."},
+                status=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            )
+        try:
+            logger.info("Extracting dataset...")
+
+            uploaded_file = request.FILES["file"]
+            fs = FileSystemStorage(location=settings.MEDIA_ROOT)
+            file_name = fs.save(uploaded_file.name, uploaded_file)
```

Comment:
No no, we can't do that!

This is completely not what DDD is used for.
I know it's not a business logic, but like that an "general" operation is escaping to a view. We can't agree on that.
This must be hidden in service command or repository.

In other words, we need to have a function with will accept `ByteIO` and return Path to file where it was saved.